example/basic/basic-rest/src/main/resources/accumulo/store.properties

```diff
@@ -14,13 +14,11 @@
 # limitations under the License.
 #
 # Accumulo store config
-gaffer.store.class=uk.gov.gchq.gaffer.accumulostore.DemoMiniAccumuloStore
+gaffer.store.class=uk.gov.gchq.gaffer.accumulostore.MockAccumuloStore
```

Comment:
Does the Mock store need to be used here?

Reply:
The reason I put it there was to maintain an Accumulo store demo. The only other option would be to have a full on Accumulo store.

Reply:
Does the MiniAccumulo store not work here?

Reply:
No. I've put it in the test jar to keep out the issues with dependencies (so that the accumulo-rest still works with an AccumuloStore) so it can be used for testing but not as a deployment.